### ISSUE TITLE
X-RAY

### ISSUE THREAD

**Issue by fhgb54643689** [studio-minus/ppg-bugs-and-requests#565]
**Feature description**
a new object in the machines tab named "X-RAY" or some other variation which would allow the player to put a human or any other entity behind it allowing you to see there bones and vital parts aka heart lungs brain etc.

**Feature purpose**
it would allow the player to have a fun way of seeing the vital parts on a human and also the general bone structure of said entity without burning it to a crisp.

**Additional context**
none.

**Comment by DindinYT37** [comment]
Make it also show internal damage and broken bones. It would be a neat way to see what kind of internal injuries an NPC is suffering from.

**Comment by atwal6400** [comment]
copy of #277

**Comment by mestiez** [comment]
> copy of #277

Yeah it seems like it. This issue is a lot clearer though so I'll close the older one.

**Comment by atwal6400** [comment]
> > copy of #277
> 
> Yeah it seems like it. This issue is a lot clearer though so I'll close the older one.

tru
